frontend/src/modules/editor/selectTheme/views/editorThemeItemView.js

```diff
@@ -24,6 +24,19 @@ define(function(require){
     },
 
     postRender: function() {
+      var previewUrl = '/api/theme/preview/' + this.model.get('name') + '/' + this.model.get('name')
+      var $previewLoc = this.$('.theme-preview');
+
+      $.ajax(previewUrl, {
+        statusCode: {
+          200: function() {
+            $previewLoc.prepend($('<img/>', { src: previewUrl, alt: Origin.l10n.t('app.themepreviewalt') }));
+          },
+          204: function() {
+            $previewLoc.prepend($('<i/>', { class: 'fa fa-file-image-o' }));
```

Comment:
That's fine by me ive asked one of our creatives to see if they know anything better than the paintbrush as im not sure its quite right.